.github/workflows/CI-test-api.yml

```diff
@@ -51,7 +22,7 @@ jobs:
           mkdir api/test/wasm
           wget -O api/test/wasm/examples.tar.gz https://github.com/gear-tech/gear/releases/download/build/examples.tar.gz
           chmod +x gear-node
-          ./gear-node --dev --tmp &
+          ./gear-node --dev --tmp & sleep 10
```

Comment:
Up to you, but maybe this could be cleaner with npm.im/wait-on or something along those lines?

Reply:
I think it is redudant here